src/commands/appdistribution-distribute.ts

```diff
@@ -72,11 +72,24 @@ module.exports = new Command("appdistribution:distribute <distribution-file>")
     const testers = getTestersOrGroups(options.testers, options.testersFile);
     const groups = getTestersOrGroups(options.groups, options.groupsFile);
     const requests = new AppDistributionClient(appId);
+    let app: AppDistributionApp;
 
     try {
-      await requests.provisionApp();
+      app = await requests.getApp();
```

Comment:
Is it necessary to check for provisioning as an explicit step, or is there a known error that would be thrown by the `getReleaseIdByHash()` API call so we can avoid an extra round trip?

Reply:
We want to ensure that a contact email is set, hence doing this as an explicit step (see the check below).

Reply:
But if provisioning isn't done, I'd assume the other API calls would fail? I'm not going to block on this, but if you actually want to make that a prerequisite you shouldn't be able to bypass it just by not making a client call :sweat_smile:

Reply:
Hmm, I suppose we actually don't because this command doesn't currently require a project (it should and will have to in the future once the APIs are project-scoped, though).

You can omit the URL for now.

Reply:
Yes, the other calls would fail, but they aren’t set up to return specific errors (app not provisioned, email not set, etc.)